command/agent/local.go

```diff
@@ -499,6 +516,8 @@ func (l *localState) syncChanges() error {
 		if err := l.syncNodeInfo(); err != nil {
 			return err
 		}
+	} else {
+		l.logger.Printf("[DEBUG] agent: Node info in sync")
```

Comment:
Cool. I was concerned this would fire per check and would be spammy, but it looks like it's called only once per AE run.